qpsolvers/solvers/scs_.py

```diff
@@ -91,12 +90,6 @@ def scs_solve_qp(
         Warm-start guess vector (not used).
```

Comment:
In the comment:

 & G x \\leq h                \\\\
  & A x = h

one of these should be 'b' I think?

Reply:
Oh my, this typo has been around for a while :sweat_smile: Great catch, thank you!

Fixed in f22176be7d775e2928c2a37ea6cd06d1e302817d.